portfolio/src/main/webapp/index.html

```diff
@@ -223,6 +224,8 @@ <h2>Have an Opinion? Add a comment below!</h2>
             <input type="text" name="form-comment" id="form-comment-input" onkeyup="configureBtn()">
             <label for="form-name-input">Name</label>
             <input type="text" name="form-name" id="form-name-input" onkeyup="configureBtn()">
+            <label for="form-email-input">Email</label>
```

Comment:
do you think it would be a bad idea to just make the display of the input none?

Reply:
so that it's hidden

Reply:
I think it would be better design to have the JS append the email to the POST request instead (you'd have to construct a POST url similar to how `fetchFromData` does it). 

To do this you can have a variable to store the email after checking user login :)